lib/rules/no-root-v-if.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @author Perry Song
+ * @copyright 2023 Perry Song. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const utils = require('../utils')
+
+/**
+ * Get the number of root element directive
+ * @param {VNode[]} rootElements The start tag node to check.
+ */
+function getDirectivesLength(rootElements) {
+  let ifLength = 0
+  let elseLength = 0
+  let elseIfLength = 0
+
+  for (const element of rootElements) {
+    if (element.type === 'VElement') {
+      if (utils.hasDirective(element, 'if')) ifLength += 1
+      if (utils.hasDirective(element, 'else')) elseLength += 1
+      if (utils.hasDirective(element, 'else-if')) elseIfLength += 1
+    }
+  }
+
+  return {
+    ifLength,
+    elseLength,
+    elseIfLength
+  }
+}
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'enforce valid `v-if` directives on root element',
```

Comment:
I don't think this description is correct.
I think that rule just reports the v-if of the root element. `valid`/`invalid` is irrelevant.